pages/_document.js

```diff
@@ -77,6 +77,9 @@ class MyDocument extends Document {
             />
           ))}
           <link rel="stylesheet" href={font} />
+          {url.match(/blog/) ? (
+            <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Slab:300" />
+          ) : null}
```

Comment:
Hm... This only works on first page load.

Reply:
-> _app.js

Reply:
Ah, okay thanks. I thought it was the other way around...